### ISSUE TITLE
Ally Map Sharing not working

### ISSUE THREAD

**Issue by Phr3d13** [FTBTeam/FTB-Chunks#94]
Minecraft version 1.16.4, forge version 35.1.36, mod versions ftb-chunks-1604.2.2.57 and ftb-gui-library-1604.1.1.26

**Comment by sarim** [comment]
looks like its disabled in https://github.com/FTBTeam/FTB-Chunks/commit/1abfdbbb1634495300ba84c94930a630467e1e0a . But its not mentioned what bug.

**Comment by MaxNeedsSnacks** [comment]
Yep, we've disabled it temporarily as it caused some problems with the new block format in FTB Chunks; it'll hopefully be reimplemented soon
